### PR TITLE
if there is no title use url for the new keep notification

### PR DESCRIPTION
**Pull request by eishay**

### CODE REVIEW

**Comment by eishay** [comment]
@leogrim

File: kifi-backend/modules/eliza/app/com/keepit/notify/info/NotificationKindInfoRequests.scala [hunk 1]
```diff
@@ -104,15 +104,13 @@ class NotificationKindInfoRequests @Inject()(implicit val pubIdConfig: PublicIdC
...
 
       val body = {
-
-
         slackAttributionOpt.map { attr =>
-          val titleString = newKeep.title.map(title => s": $title") getOrElse("")
+          val titleString = newKeep.title.getOrElse(newKeep.url)
           attr.message.channel.name match {
-            case Some(prettyChannelName) => s"${author.name} just added in #$prettyChannelName" + titleString
+            case Some(prettyChannelName) => s"${author.name} just added in #${prettyChannelName.value}: " + titleString
```
Comment (eishay) [inline kifi-backend/modules/eliza/app/com/keepit/notify/info/NotificationKindInfoRequests.scala:110]: thanks for @ryanpbrewster to show me that one
https://www.evernote.com/l/ABL-EhKUAy5IA4yU3GH8tY7DagdEZjRiJjMB/image.png

File: kifi-backend/modules/eliza/app/com/keepit/notify/info/NotificationKindInfoRequests.scala [hunk 2]
```diff
@@ -104,15 +104,13 @@ class NotificationKindInfoRequests @Inject()(implicit val pubIdConfig: PublicIdC
...
       val body = {
-
-
         slackAttributionOpt.map { attr =>
-          val titleString = newKeep.title.map(title => s": $title") getOrElse("")
+          val titleString = newKeep.title.getOrElse(newKeep.url)
           attr.message.channel.name match {
-            case Some(prettyChannelName) => s"${author.name} just added in #$prettyChannelName" + titleString
+            case Some(prettyChannelName) => s"${author.name} just added in #$prettyChannelName: " + titleString
             case None => s"${author.name} just shared" + titleString
```
Comment (leogrim) [inline kifi-backend/modules/eliza/app/com/keepit/notify/info/NotificationKindInfoRequests.scala:111]: I think you need a space in these

File: kifi-backend/modules/eliza/app/com/keepit/notify/info/NotificationKindInfoRequests.scala [hunk 3]
```diff
@@ -104,15 +104,13 @@ class NotificationKindInfoRequests @Inject()(implicit val pubIdConfig: PublicIdC
       val slackAttributionOpt = newKeep.attribution
 
       val body = {
-
-
         slackAttributionOpt.map { attr =>
-          val titleString = newKeep.title.map(title => s": $title") getOrElse("")
+          val titleString = newKeep.title.getOrElse(newKeep.url)
```
Comment (leogrim) [inline kifi-backend/modules/eliza/app/com/keepit/notify/info/NotificationKindInfoRequests.scala:108]: You can move this one out of the scope now, to use in the `getOrElse`

**Pull request merged**